modules/custom_operations/user_ie_extensions/ov_extension.cpp

```diff
@@ -86,6 +86,15 @@
 #    define TOKENIZER_EXT
 #endif
 
+#ifdef paged_attention
+#    include "paged_attention/paged_attention.hpp"
+#    define PAGED_ATTENTION_EXT \
+            std::make_shared<ov::OpExtension<TemplateExtension::PagedAttention>>(), \
+            std::make_shared<ov::frontend::OpExtension<TemplateExtension::PagedAttention>>(),
```

Comment:
With this line, do we still need https://github.com/openvinotoolkit/openvino/pull/22656?